packages/playwright/src/reporter.ts

```diff
@@ -327,9 +340,21 @@ class ReplayPlaywrightReporter implements Reporter {
   async onEnd() {
     try {
       await this.reporter.onEnd();
-      if (!this._foundReplayBrowser) {
+
+      const projectsWithoutReplay = Object.keys(this._projects).filter(projectName => {
+        const { executed, usingReplay } = this._projects[projectName];
+        return executed && !usingReplay;
+      });
+      if (projectsWithoutReplay.length) {
+        const projectText =
+          projectsWithoutReplay[0] === ROOT_PROJECT_NAME
+            ? "Your project"
+            : `${projectsWithoutReplay.join(", ")} project${
+                projectsWithoutReplay.length > 1 ? "s" : ""
+              }`;
+
```

Comment:
I'm not sure this produces a meaningfully better warning message to be worth it. I think just saying "Your tests ran without Replay Chromium" would be sufficient.

What would this actually look like in practice? My experience is pretty limited here, but the way I've typically seen projects named were like "firefox" or "chrome" So in that case this would print a warning like:

> chrome project ran without Replay Chromium. If this wasn't intentional, please recheck your configuration.

or

> firefox, chrome projects ran without Replay Chromium. If this wasn't intentional, please recheck your configuration.

Both of those seem kind of awkward to me, particularly if the project has a 1:1 relationship with a browser. I guess in most cases it would be the root project so we'd just say "Your project ran without Replay Chromium" but in that case, I think that saying "Your tests ran without Replay Chromium" might still read a little better?